lib/utils/utils.js

```diff
@@ -0,0 +1,13 @@
+export function unpublish (version) {
+  return {
+    version,
+    actions: [{ action: 'unpublish' }]
+  }
+}
+
+export function publish (version) {
```

Comment:
This `utils/utils/publish` function is not used anywhere in the code

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/30/commits/7016cd557776114b58a5bc364ae2285bfb80f5a1